modernizer-annotation-processor/src/main/java/org/gaul/modernizer_annotation_processor/ModernizerAnnotationProcessor.java

```diff
@@ -217,7 +218,7 @@ private String getClassName(Element classElement) {
     }
 
     private String getMethodRepresentation(Element methodElement) {
-        ExecutableType method = getExecutableTypeElement(methodElement);
```

Comment:
Can you delete `getExecutableTypeElement` now?

Reply:
Yea pushing it in a different commit.

Reply:
Huh, shouldn't they be at the same version?

Reply:
I think you meant to reply on the previous comment. Anyway, I'm not sure how they are on different versions even though they have the same parent module. In Maven Central, different versions are shown -
https://mvnrepository.com/artifact/com.google.auto.service/auto-service
https://mvnrepository.com/artifact/com.google.auto/auto-common